Utils/github_workflow_scripts/check_if_needs_to_fill_contribution_form.py

```diff
@@ -0,0 +1,146 @@
+
+    print(f'Did not find a {METADATA} file in the PR.')
+    return ''
+
+
+def get_pack_support_type_from_pr_metadata_file(pr_metadata_filename: str, pr: PullRequest):
+    """ Retrieves the support type from the pr metadata.json file
+
+    Args:
+        pr_metadata_filename: The pr metadata.json filename
+        pr: The pr
+
+    Returns:
+        The support type
+
+    """
+    print(f'Getting support type from {pr_metadata_filename}.')
+    _, branch_name = pr.head.label.split(':')
+    print(f'Branch name is: {branch_name}')
+    contributor_repo: Repository = pr.head.repo
+    branch: Branch = contributor_repo.get_branch(branch=branch_name)
+    metadata_file: ContentFile = contributor_repo.get_contents(path=pr_metadata_filename, ref=branch.commit.sha)
+    metadata_file_content: dict = json.loads(base64.b64decode(metadata_file.content))
+    return metadata_file_content.get(SUPPORT)
+
+
+def get_pack_name_from_pr(pr_files) -> str:
+    """ Extracts the pack name from the pr files
+
+    Args:
+        pr_files (PaginatedList[File]): The list of pr files
+
+    Returns:
+        The pack name
+
+    """
+    for file in pr_files:
+        if PACKS in file.filename:
+            return re.findall(PACK_NAME_REGEX, file.filename)[0]
+
+    raise Exception('PR does not contains files prefixed with "Packs".')
+
+
+def get_pack_support_type_from_repo_metadata_file(pack_name):
+    """ Retrieves the support type from the repo metadata.json file
+
+    Args:
+        pack_name (str): The pack name
+
+    Returns:
+        The support type
+
+    """
+    print(f'Getting support type from the repo.')
+    repo_pack_metadata_path: str = os.path.join(CONTENT_ROOT_PATH, PACKS, pack_name, METADATA)
+    print(f'Pack {METADATA} file is at path: {repo_pack_metadata_path}')
+    repo_pack_metadata: dict = load_json(repo_pack_metadata_path)
+    return repo_pack_metadata.get(SUPPORT)
+
+
+def arguments_handler():
+    """ Validates and parses script arguments.
+
+     Returns:
+        Namespace: Parsed arguments object.
+
+     """
+    parser = argparse.ArgumentParser(description='Check if the contribution form needs to be filled.')
+    parser.add_argument('-p', '--pr_number', help='The PR number to check if the contribution form needs to be filled.')
+    parser.add_argument('-g', '--github_token', help='The GitHub token to authenticate the GitHub client.')
+    return parser.parse_args()
+
+
+def main():
+    options = arguments_handler()
+    pr_number = options.pr_number
+    github_token = options.github_token
+
+    org_name: str = 'demisto'
+    repo_name: str = 'content'
+    github_client: Github = Github(github_token, verify=False)
+    content_repo: Repository = github_client.get_repo(f'{org_name}/{repo_name}')
+    pr: PullRequest = content_repo.get_pull(int(pr_number))
+    pr_files = pr.get_files()
+    pack_name: str = get_pack_name_from_pr(pr_files)
+
+    if pr_metadata_filename := get_metadata_filename_from_pr(pr_files):
```

Comment:
Need to consider the case that there is more than one pack in the PR. The code should iterate on all and check for all support types, if there is at least one that is not XSOAR then fail.